templates/ros2_control/controller/dummy_chainable_controller.cpp

```diff
@@ -93,13 +95,15 @@ controller_interface::CallbackReturn DummyClassName::on_configure(
   subscribers_qos.keep_last(1);
   subscribers_qos.best_effort();
 
+  ref_timeout_ = rclcpp::Duration::from_seconds(params_.reference_timeout);
+
   // Reference Subscriber
   ref_subscriber_ = get_node()->create_subscription<ControllerReferenceMsg>(
-    "~/reference", subscribers_qos,
+    "~/commands", subscribers_qos,
```

Comment:
```suggestion
    "~/reference", subscribers_qos,
```

Reply:
ok change ing it to /reference everywhere